tests/bento_service_examples/fasttext_classifier.py

```diff
@@ -0,0 +1,13 @@
+import bentoml
+import json
+from bentoml.adapters import JsonInput
+from bentoml.frameworks.fasttext import FasttextModelArtifact
+
+
+@bentoml.env(infer_pip_packages=True)
+@bentoml.artifacts([FasttextModelArtifact('model')])
+class FasttextClassifier(bentoml.BentoService):
+    @bentoml.api(input=JsonInput(), batch=False)
+    def predict(self, parsed_json):
+        # parsed_json is of type str - make it a dict
```

Comment:
Check out [this similar test implementation](https://github.com/bentoml/BentoML/blob/5450d6cddacc04e37be4daaedb4b419c65c821cc/tests/bento_service_examples/transformer_gpt_example.py) and the [corresponding test](https://github.com/bentoml/BentoML/blob/5450d6cddacc04e37be4daaedb4b419c65c821cc/tests/integration/test_transformers_model_artifact.py). I think something else is causing `parsed_json` to be a string.